src/plugins/controls/public/react_controls/control_group/init_controls_manager.ts

```diff
@@ -222,3 +234,13 @@ export function initControlsManager(initialControlPanelsState: ControlPanelsStat
     >,
   };
 }
+
+function getLastDataViewId(initialControlPanelsState: ControlPanelsState) {
```

Comment:
The current controls have logic in place to grab the first data view ID from the parent when there are 0 controls. This makes creating the first control much faster on Dashboards that already have at least one panel with a data view, and will also implicitly fall back to the default data view when the parent has no panels that publish data views.

This can be accomplished with something like

``` typescript
if (Object.keys(initialControlPanelsState).length === 0 && apiPublishesDataViews(api.parent)) {
  dataViewId = api.parent.dataViews[0]?.id;
}
```